beetsplug/subsonicupdate.py

```diff
@@ -88,26 +90,35 @@ def __format_url():
                 context_path = ''
             url = "http://{}:{}{}".format(host, port, context_path)
 
-        return url + '/rest/startScan'
+        return url + '/rest/startScan.view'
```

Comment:
Is this valid for all versions of subsonic?

Reply:
I have test with  http://demo.subsonic.org/ seem works

Reply:
Ahh, it's ok! Luckily it works in both. I didn't realize this was also another deprecated API thing.

Thank you!